.gitignore

```diff
@@ -1,3 +1,4 @@
+.cache
```

Comment:
it seems to in this case. This will ignore all directories named `.cache` so it will work either way.